src/util.rs

```diff
@@ -110,7 +110,7 @@ pub fn serialize(data: &[Field]) -> Vec<u8> {
 
 /// Get a vector of field elements from a byte slice
 pub fn deserialize(data: &[u8]) -> Vec<Field> {
-    let field_size = std::mem::size_of::<Field>();
+    let field_size = 4;
```

Comment:
Was `std::mem::size_of` not working? While I don't expect the width of `u32` to change I'd still prefer not to hard code this.

Reply:
I added a function `size()` to `Fp` that returns the number of bytes needed to encode field elements as integers and replaced the hard-coded value with `SMALL_FP.size()`. Here and in prng.rs.